src/logic/Operate.js

```diff
@@ -0,0 +1,14 @@
+import Big from 'big';
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const operations = {
+    '+': (n1, n2) => Big(n1) + Big(n2),
+    '-': (n1, n2) => Big(n1) - Big(n2),
+    x: (n1, n2) => Big(n1) * Big(n2),
+    '÷': (n1, n2) => Big(n1) / Big(n2),
+  };
+
+  operations[operation](numberOne, numberTwo);
+};
```

Comment:
wise move

Reply:
anticipate division by Zero error for the division-operation (`5/0`). This will return an error in the next milestone. You might have well work on that scenario now.